float64 current_40
float64 voltage_40

float64 current_41
float64 voltage_41

float64 current_44
float64 voltage_44